Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -843,14 +843,26 @@ extension ProjectDescription.ResourceFileElements {
         // Add default resources path if necessary
         // They are handled like a `.process` rule
         if sources == nil {
-            resourceFileElements += try defaultResourcePaths(from: path)
+            // Already included resources should not be added as default resource
+            let excludedPaths: Set<AbsolutePath> = Set(
```

Comment:
I don't really know what the best approach is here, but basically any file already included by an explicit resource file element should not be included one more time as a default resource.